### DISCUSSION TITLE
Decrease desktop CSS threshold

### DISCUSSION THREAD

**Discussion by saloniamatteo** [searxng/searxng#4014] [Q&A] [answered]
Greetings everyone,
would it be possible to lower the CSS width threshold upon which the "desktop" layout is loaded?

Currently, the width threshold is set to `@media screen and (max-width: 79.75em)`, which equals roughly `1276px`: what this means is, devices with a width of `800px < w < 1276px` will load the tablet (?) layout:

![Screenshot_20241114_143339](https://github.com/user-attachments/assets/d4ea5a0a-8b87-4304-89f1-c2d679560ea0)

Devices width a width of `w > 1276px` will load the desktop layout:

![Screenshot_20241114_143400](https://github.com/user-attachments/assets/fcbed6b9-785e-47c5-902a-3aff1434b434)

This makes it particularly weird for those users (including me) that slightly resize the window, even when having a bigger monitor with two windows open side-by-side.

Would it be possible, then, to lower this threshold, from 1276px (79.75em), to a value around 1160px (72.5em)? This is particularly useful on laptops.

----

Also, I'm running my own SearXNG instance, it'd be useful to know how I can modify SearXNG's theme (e.g. having a file with CSS overrides).

**Comment by return42** [comment, marked as answer]
> (e.g. having a file with CSS overrides).

SearXNG has no option for a custom theme.

> Would it be possible, then, to lower this threshold, from 1276px (79.75em), to a value around 1160px (72.5em)? This is particularly useful on laptops.

The size was discussed / implemented here:

- https://github.com/searxng/searxng/issues/874
- https://github.com/searxng/searxng/pull/943

If you want, we can convert this discussion into a request .. if you are not alone, than we can consider to modify the device sizes: 

https://github.com/searxng/searxng/blob/10d3af84b833ab2f2d1095efa3a7ba240ffb32fc/searx/static/themes/simple/src/less/definitions.less#L285-L290

**Reply by saloniamatteo** [reply]
Sure, if you recommend so, then let's convert this discussion into a request :)
